extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -232,51 +228,51 @@ $color-rail-background: mix($color-page, $black, 90%);
 
                 .carousel-container {
                     left: 16px;
-                    width: 260px;
+                    width: 220px;
                 }
 
                 .since, .chatter {
-                    width: 32px;
+                    width: 30px;
                 }
+            }
+        }
+    }
+}
 
-                .UserStatsMenu {
-                    outline: rgba($color-divider, .3) solid 2px;
-                    width: 220px;
+.UserStatsMenu .new-chat {
+    outline: rgba($color-divider, .3) solid 2px;
+    width: 220px;
 
-                    .actions ul {
-                        margin: 0;
-
-                        li {
-                            border-bottom: $color-divider solid thin;
-                            cursor: pointer;
-                            height: 44px;
-                            margin: 0;
-                            white-space: nowrap;
-
-                            .label {
-                                color: mix($color-text, $color-links, 80%);
-                                line-height: 44px;
-                                padding-left: 10px;
-                            }
-
-                            &:hover {
-                                background-color: $color-buttons;
-                            }
-                        }
-                    }
+    .actions ul {
```

Comment:
the owner .actions class is DIV which containts ul. We do not want to style DIV, but it's internal element (ul).